db/migrate/20220905152155_create_recipes.rb

```diff
@@ -0,0 +1,14 @@
+class CreateRecipes < ActiveRecord::Migration[7.0]
+  def change
+    create_table :recipes do |t|
+      t.string :name
+      t.string :preparation_time
+      t.string :cooking_time
+      t.text :description
+      t.boolean :public
+      t.references :user, null: false, foreign_key: true
+
+      t.timestamps
+    end
+  end
+end
```

Comment:
- [ ] Please make sure that the migrations run successfully for your project to be reviewed correctly. Currently, the migrations fail and produce the following error (See attached below). To see this error, you need to drop your DB, create it, and then run the migrations and you should be able to see this error.

- [HINT] This error happens because the first migration you have is for creating the recipe table, in which you add the reference for the users' table which hasn't been created yet. If you fix this error, you'll be presented with an error in another file produced by the same reason. All you need to do is to make sure the order of your migrations is correct and tables don't use references for another table that hasn't been created yet.

![image](https://user-images.githubusercontent.com/62290545/195637438-cbe6efab-e17d-446a-99c1-f77fb9798b7c.png)

Reply:
This change has been done well